spec/requests/api/v3/providers/courses/index_spec.rb

```diff
@@ -87,6 +87,7 @@ def perform_request
                 "is_send?" => true,
                 "level" => "primary",
                 "applications_open_from" => provider.courses[0].applications_open_from.strftime("%Y-%m-%d"),
+                "provider_type" => nil,
```

Comment:
There are a couple of providers with `provider_type == nil` but what does that mean? Should we make sure there's a value in the tests?

Reply:
yes need to look into that...

Reply:
https://github.com/DFE-Digital/teacher-training-api/pull/1440